docs/configuration.md

```diff
@@ -1796,6 +1801,51 @@ defines a such a feature with configurable behavior.
   </tr>
 </table>
 
+## Validation
+
+    validation:
+      manifests:
+        urls:
+          allowed: ^https?://(www\.)?example\.com
+          disabled: false
+
+### Manifests
+
+Use the `manifest` subsection to configure manifest validation.
+
+#### URLs
```

Comment:
Make this clear that this applies to any descriptors that have urls in a manifest. This validation needs to include any descriptor in manifest list, as well.